css/apigee_m10n.future-links-field.css

```diff
@@ -0,0 +1,21 @@
+.field--name-futureplanlinks {
```

Comment:
The reason they are separate is because the CSS in each is specific to a field which may or may not be displayed. CSS aggregation in Drupal 8 handles this kind of thing really well.

Reply:
Sounds good